.travis.yml

```diff
@@ -111,3 +124,18 @@ script:
     else
       ctest -V
     fi
+  - |
+    if [[ "${DOCUMENTATION}" == "true" ]]; then
```

Comment:
I think we probably only want to deploy this on success, so the deployment part of this should probably go to the `after_success:` hook (see https://docs.travis-ci.com/user/deployment/custom/)